docs/submitted_reports.md

```diff
@@ -85,3 +85,36 @@ log-[1-7].log
 ```
 
 Log 1-7 are logs from a round-robin buffer and are ordered but the start point is undefined
+
+## Third Room
+
+Third Room uses `logs.json` file to submit logs.
+
+`logs.json` file format:
+```
+{
+  "formatVersion": 1,
+  "appVersion": "0.0.0",
+  "platform": "<UserAgent Info>",
+  "items": [LogItem, LogItem, ...]
+}
+```
+
+`LogItem` interface:
+```typescript
+interface LogItem {
+  s: number; // start time
+  d?: number; // duration time
+  v: { // log item value
+    [key: string]: any;
+  };
+  l: number; // log level
+  e?: { // error
+      stack?: string;
+      name: string;
+      message: string;
+  };
+  f?: boolean; // forced
+  c?: LogItem[]; // children
+}
+```
```

Comment:
I don't find this particularly easy to understand (also it assumes knowledge of typescript syntax, which not all readers of this file may know).

Could you maybe write it out as a markdown list, and expand on the meaning of the properties

```markdown
Each `LogItem` is an object with the following properties:

 * `s`: start time XXX what units?
 * `d`: duration XXX of what? When is this present? When is it not?
```

etc